CHANGES.rst

```diff
@@ -9,6 +9,7 @@ Version 0.8.0 (UNRELEASED)
 - Adds job start and finish times to ``logs`` command output.
 - Changes ``delete`` to prevent workflow hard deletion.
 - Changes ``list`` to display interactive session status.
+- Adds ``--include-progress`` and ``--include-workspace-size`` in ``list`` command.
```

Comment:
Could you please be more explicit? As a user, I might not know what these two flags refer to. I would expect some minimum explanation on why those flags were introduced.

```
- Changes ``list`` command to stop including workflow progress and workspace size by default. Adds ``--include-progress`` and ``--include-workspace-size`` options to add them to the output.